docs/Contributing/API-for-contributors.md

```diff
@@ -2316,6 +2316,24 @@ Same as [Get host's mobile device management and Munki information](https://flee
 | ----- | ------ | ---- | ---------------------------------- |
 | token | string | path | The device's authentication token. |
 
+#### Ping Server with Device Token
+Ping the server. OK response expected if the device token is still valid.
+
+`HEAD /api/v1/fleet/device/{token}/ping`
```

Comment:
Double checking: What will happen with Orbits talking to Fleet that doesn't have this endpoint implemented? 
https://github.com/fleetdm/fleet/blob/main/docs/Contributing/fleetd-development-and-release-strategy.md

Reply:
Oh I just saw the checking below, I believe we are good.